indexer/covidResearchPaperConnector/covid.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
we can have a better name for this file:
covid_papers_schema.json

Reply:
done